ruuvitag_sensor/data_formats.py

```diff
@@ -103,6 +103,9 @@ def convert_data(raw: str) -> DataFormatAndRawSensorData:  # noqa: PLR0911
         if candidate.startswith("FF990406"):
             return (6, candidate[6:])
 
+        if candidate.startswith("FF9904E1"):
```

Comment:
Received data had e1 in lowercase (`FF9904e1116f....`). I can check in a separate PR how to make all these checks in the whole codebase case insensitive.
```
Found candidate FF9904e1116f48d0c63400090013001a001e031c0f00ffffffffffff0034ecb8ffffffffffd9380eab71d4
```